packages/google_maps_flutter/google_maps_flutter_ios/example/ios12/ios/RunnerTests/FLTTileProviderControllerTests.m

```diff
@@ -0,0 +1,38 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import XCTest;
+@import GoogleMaps;
+@import google_maps_flutter_ios;
+
+#import <OCMock/OCMock.h>
+
+@interface FLTTileProviderController ()
```

Comment:
Why is this necessary? `FLTTileProviderController` is publicly declared to inherit from `GMSTileLayer`, and [this is a public method on that class](https://developers.google.com/maps/documentation/ios-sdk/reference/interface_g_m_s_tile_layer).

You've described and implement this as testing a private method, but it should not actually be private.

Reply:
Done, c++ brain rot